app/javascript/pages/account/account-settings.tsx

```diff
@@ -123,35 +124,54 @@ const PasswordSection = ({ user, setUser }: SectionProps) => {
   )
 }
 
+const updateNameOrDOB = async (
+  newUser: User,
+  saveProfile: (profile: User) => void,
+  setUser: React.Dispatch<User>,
+  setLoading: React.Dispatch<boolean>
+) => {
+  return apiUpdateNameOrDOB(newUser)
+    .then((profile) => {
+      saveProfile(profile)
+      setUser(newUser)
+    })
+    .catch((error) => {
+      // TODO: In the case that a user's DOB is invalid, this is a snippet of the AxiosError that will be returned
```

Comment:
Can we include the logic now or does this have to wait for the future error banner ticket?

Reply:
The banner logic will be implemented in a future ticket